com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs

```diff
@@ -12,27 +12,13 @@ public override void OnInspectorGUI()
         {
             base.OnInspectorGUI();
 
-            Rigidbody rigidbody = (Rigidbody)target;
-            if (EditorApplication.isPlaying)
-            {
-                EditorGUI.BeginDisabledGroup(true);
-                EditorGUILayout.Vector3Field("Center Of Mass", rigidbody.centerOfMass);
-                EditorGUILayout.Vector3Field("Inertia Tensor", rigidbody.inertiaTensor);
-                EditorGUILayout.Vector3Field("Inertia Tensor Rotation", rigidbody.inertiaTensorRotation.eulerAngles);
-                EditorGUI.EndDisabledGroup();
-            }
-            else
-            {
-                rigidbody.centerOfMass = EditorGUILayout.Vector3Field("Center Of Mass", rigidbody.centerOfMass);
-                rigidbody.inertiaTensor = EditorGUILayout.Vector3Field("Inertia Tensor", rigidbody.inertiaTensor);
+            Rigidbody _rigidbody = (Rigidbody)target;
+            _rigidbody.centerOfMass = EditorGUILayout.Vector3Field("Center Of Mass", _rigidbody.centerOfMass);
+            _rigidbody.inertiaTensor = EditorGUILayout.Vector3Field("Inertia Tensor", _rigidbody.inertiaTensor);
 
-                Vector3 inertiaTensorEuler = rigidbody.inertiaTensorRotation.eulerAngles;
-                Vector3 newEuler = EditorGUILayout.Vector3Field("Inertia Tensor Rotation", inertiaTensorEuler);
-                if (inertiaTensorEuler != newEuler)
-                {
-                    rigidbody.inertiaTensorRotation = Quaternion.Euler(newEuler);
-                }
-            }
+            Quaternion inertiaTensorRotation = new Quaternion();
+            inertiaTensorRotation.eulerAngles = EditorGUILayout.Vector3Field("Inertia Tensor Rotation", _rigidbody.inertiaTensorRotation.eulerAngles);
+            _rigidbody.inertiaTensorRotation = inertiaTensorRotation;
```

Comment:
Is this entire code block intentionally removed?

Reply:
This is what I don't know it is removed automatically when merging main to dev

Reply:
This should be removed. This was changed against this issue.
https://github.com/Unity-Technologies/URDF-Importer/issues/59